src/issues/pages/IssueChat.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <div
+    class="relative-position"
+    v-if="issue"
+  >
+    <QCollapsible
+      opened
+      class="bg-grey-2"
+    >
+      <template slot="header">
+        <b>{{ $t('CONFLICT.INITIAL') }}</b>
+      </template>
+      <div class="q-mx-sm q-mb-sm q-pa-sm">
+        <span class="text-bold text-primary">
+          {{ $t('CONFLICT.WITH', { userName: issue.affectedUser.displayName }) }}
+        </span>
+        <ProfilePicture
+          :user="issue.affectedUser"
+          :size="25"
+          class="q-mt-xs"
+        />
+      </div>
+      <div class="q-mx-sm q-mb-sm q-pa-sm bg-white">
+        <span class="text-bold text-secondary uppercase">
+          <RouterLink
+            place="userName"
+            @click.native.stop
+            :to="{name: 'user', params: { userId: issue.createdBy.id }}"
+          >
+            {{ issue.createdBy.displayName }}
+          </RouterLink>
+        </span>
+        <span class="message-date">
+          <small class="text-weight-light">
+            <DateAsWords :date="issue.createdAt" />
+          </small>
+        </span>
+        <Markdown :source="issue.topic" />
+      </div>
+      <div
+        v-if="conversation"
+        class="q-ml-sm q-pt-sm q-pl-sm"
+      >
+        <div>
+          <ProfilePicture
+            v-for="user in conversation.participants"
+            :key="user.id"
+            :user="user"
+            :size="20"
+            class="q-mr-xs"
+          />
+        </div>
+        <div class="q-caption q-caption-opacity q-mt-xs">
+          {{ $t('ISSUE.PARTICIPANTS', { count: conversation.participants.length }) }}
+        </div>
+      </div>
+    </QCollapsible>
+    <div
+      v-if="conversation"
+      class="bg-secondary absolute-top-right round-borders q-pa-xs"
+    >
+      <NotificationToggle
+        :muted="conversation.muted"
+        :is-participant="conversation.isParticipant"
+        :user="currentUser"
+        in-toolbar
+        @set.stop="setNotifications"
+        :size="$q.platform.is.mobile ? 'sm' : 'md'"
+      />
```

Comment:
Also, setting notifications to off means I leave the conversation, but I cannot rejoin it again :/

I think it would be reasonable to be a member of the conversation but have all notifications off (distinct from not being a member of the conversation - but with same notification behaviour).

Reply:
As a short-term fix, I'll remove the "off" option for issue chats. 

I'm less motivated right now to implement yet another notification state, but maybe it's necessary at some point.

Reply:
Ah, I guess I should really fix this in the backend... let's see what the best option is.